akka-http/src/main/scala/akka/http/scaladsl/server/RouteResult.scala

```diff
@@ -27,6 +29,7 @@ object RouteResult {
                                                routingLog: RoutingLog,
                                                executionContext: ExecutionContext = null,
                                                rejectionHandler: RejectionHandler = RejectionHandler.default,
-                                               exceptionHandler: ExceptionHandler = null): Flow[HttpRequest, HttpResponse, Unit] =
+                                               exceptionHandler: ExceptionHandler = null,
+                                               parserSettings: ParserSettings): Flow[HttpRequest, HttpResponse, Unit] =
```

Comment:
I'd rather not pass it around as implicit I think - can you remove it from the implicit argument list there and pass around explicitly? Thanks!